src/editor/options.js

```diff
@@ -12,7 +12,11 @@ const fitInfo = document.querySelector('.info--fit');
 export function updatePreview(input) {
   if (input.className !== 'control__input') return;
   const preview = /** @type {HTMLSpanElement} */ (input.nextElementSibling);
-  preview.textContent = input.value + (preview.dataset.suffix || '');
+   if (input.name === 'fill') {
```

Comment:
If we check if the preview is an input, it will fix the typescript errors and allow for other controls to use text inputs in the future

```suggestion
   const preview = /** @type {HTMLSpanElement | HTMLInputElement} */ (input.nextElementSibling);
   if (preview instanceof HTMLInputElement) {
```

Reply:
Thank you @NotWoods. Actually I'm not confortable with typescript, this help me a lot.